config/routes.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  devise_for :users, controllers: { confirmations: 'users/confirmations',
+                                    passwords: 'users/passwords',
+                                    registrations: 'users/registrations', sessions: 'users/sessions' }
+
+  devise_scope :user do
+    get '/users/sign_out', to: 'devise/sessions#destroy'
+    get '/users/password', to: 'users/passwords#create'
+    get '/users/sessions', to: 'users/sessions#index'
+    root 'users/sessions#index'
+  end
+
+
+
+  resources :groups do
+    resources :budgets
+  end
```

Comment:
- [ ] @J2ZROMERO, we really need to make sure that only necessary routes are used on this project.  By default, Rails creates routes for all seven of the default actions (index, show, new, create, edit, update, and destroy) for every RESTful route in your application. You can use the :only and :except options to fine-tune this behavior. The **:only** option specifies that only certain routes should be generated. This can really save our memory space



@J2ZROMERO you are doing well in this project :+1:

Reply:
Done!